### PR TITLE
Multibrand UI updates

### PR DESCRIPTION
**Pull request by brentschaus-okta**
## Description:
- **What's changed?** 
- **Is this PR related to a Monolith release?** 

### Netlify links:
- [Custom domain and email domain](https://63e29d56a43f1320e39a5dc0--reverent-murdock-829d24.netlify.app/docs/guides/custom-url-domain/main/)
- [Sign-in page](https://63e29d56a43f1320e39a5dc0--reverent-murdock-829d24.netlify.app/docs/guides/custom-widget/main/)
- [Error pages](https://63e29d56a43f1320e39a5dc0--reverent-murdock-829d24.netlify.app/docs/guides/custom-error-pages/main/)
- [Email notifications](https://63e29d56a43f1320e39a5dc0--reverent-murdock-829d24.netlify.app/docs/guides/custom-email/main/)

### Resolves:

* [OKTA-XXXXXX](https://oktainc.atlassian.net/browse/OKTA-XXXXXX)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/custom-url-domain/main/index.md [hunk 1]
```diff
@@ -46,6 +48,12 @@ Okta serves pages on your custom domain over HTTPS. To set up this feature, you
 
 You can also [configure a custom email address](#about-custom-email-addresses) to present a branded experience to your end users.
 
+### Multibrand and custom domains <ApiLifecycle access="ea" />
+
+You can create up to three custom domains with multibrand customizations and up to 200 custom domains by contacting support to increase your limit.
+
+You can only visit a branded touchpoint (such as a logo, color, favicon, or an image applied to a page or email) once you map to a custom domain. After you create a brand, map it to a custom domain. Then you can make further customizations, preview them, and publish them.
```
Comment (albertchen-okta) [inline packages/@okta/vuepress-site/docs/guides/custom-url-domain/main/index.md:55]: You can only preview or visit a branded page (such as viewing brand assets applied to the Okta-hosted sign-in page) once you map....

File: packages/@okta/vuepress-site/docs/guides/custom-url-domain/main/index.md [hunk 2]
```diff
@@ -291,25 +301,21 @@ A custom email address allows you to present a branded experience to your end us
...
-1. In the Admin Console, go to **Customizations** > **Emails**.
+> **Note:** If you've enabled [Early Access (EA) multibrand customization](https://help.okta.com/okta_help.htm?type=oie&id=csh-branding), your Admin Console navigation is different. See parenthetical notes.
 
-2. Click the **Sender:** link (**Okta <noreply@okta.com>**) near the top of the page.
-
-3. On the Configure Email Sender dialog box, select **Custom email domain** as the type of sender that you want to send system notification emails from.
+## Configure a custom email address
 
-4. In the **Email address to send from**, enter the email address that you want to send the system notification emails from. This is what displays in the emails sent to your users.
+1. In the Admin Console, go to **Customizations** > **Emails**. (EA users: go to **Customizations** > **Brands**, and then select the brand you want. In the **Domains** tab, in the **Email domain** section, click **Add domain**).
```
Comment (albertchen-okta) [inline packages/@okta/vuepress-site/docs/guides/custom-url-domain/main/index.md:308]: in the Email section, click Edit, then Add custom email domain

File: packages/@okta/vuepress-site/docs/guides/custom-widget/main/index.md [hunk 3]
```diff
@@ -17,7 +17,8 @@ This guide explains how to customize the sign-in page for both redirect and embe
 
 * [Okta Developer Edition organization](https://developer.okta.com/signup)
 * (Redirect authentication): [Custom URL domain](/docs/guides/custom-url-domain/main/)
-* (Redirect authentication): The full-featured code editor enabled in your org <ApiLifecycle access="ea" />
+* (Redirect authentication): The full-featured code editor enabled in your org
```
Comment (albertchen-okta) [inline packages/@okta/vuepress-site/docs/guides/custom-widget/main/index.md:20]: GA as of Feb 16, same as first comment

**Review by albertchen-okta — APPROVED** [review]
a few small comments, approved otherwise

File: packages/@okta/vuepress-site/docs/guides/custom-url-domain/main/index.md (hunk 1 above)
Comment (brentschaus-okta) [reply packages/@okta/vuepress-site/docs/guides/custom-url-domain/main/index.md:55]: done.

File: packages/@okta/vuepress-site/docs/guides/custom-url-domain/main/index.md (hunk 2 above)
Comment (brentschaus-okta) [reply packages/@okta/vuepress-site/docs/guides/custom-url-domain/main/index.md:308]: done

File: packages/@okta/vuepress-site/docs/guides/custom-email/main/index.md [hunk 4]
```diff
@@ -16,7 +16,8 @@ Customize email notifications.
 
 - [Okta Developer Edition organization](https://developer.okta.com/signup)
 - Access to email template customization. Contact [Okta support](https://support.okta.com/help) for help.
-* The full-featured code editor enabled in your org <ApiLifecycle access="ea" />
+* The full-featured code editor enabled in your org
```
[the remark this answers is not in the document]
Comment (brentschaus-okta) [reply packages/@okta/vuepress-site/docs/guides/custom-email/main/index.md:19]: removed.

File: packages/@okta/vuepress-site/docs/guides/custom-widget/main/index.md (hunk 3 above)
Comment (brentschaus-okta) [reply packages/@okta/vuepress-site/docs/guides/custom-widget/main/index.md:20]: removed.

File: packages/@okta/vuepress-site/docs/guides/custom-error-pages/main/index.md [hunk 5]
```diff
@@ -40,8 +40,10 @@ The [Brands API](/docs/reference/api/brands/) allows you to set icons, images, a
 
 Use the code editor to modify any HTML, CSS, or JavaScript on the error page. See [Customization examples](#customization-examples) for snippets that you can update and use.
 
-1. In the Admin Console, go to **Customizations** > **Branding**.
-2. In the **Error Page** box, click **Edit**.
+> **Note:** If you've enabled [Early Access (EA) multibrand customization](https://help.okta.com/okta_help.htm?type=oie&id=csh-branding), your Admin Console navigation is different. See parenthetical notes.
+
+1. In the Admin Console, go to **Customizations** > **Branding**. (EA users: go to **Customizations** > **Brands**, and then select the brand you want.)
+2. In the **Error Page** box, click **Edit**. (EA user: on the **Pages** tab in the Error Pages section, click **Configure**.)
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/custom-error-pages/main/index.md:46]:
```suggestion
2. In the **Error Page** box, click **Edit**. (EA users: on the **Pages** tab in the **Error Pages** section, click **Configure**.)
```

File: packages/@okta/vuepress-site/docs/guides/custom-url-domain/main/index.md [hunk 6]
```diff
@@ -46,6 +48,12 @@ Okta serves pages on your custom domain over HTTPS. To set up this feature, you
 
 You can also [configure a custom email address](#about-custom-email-addresses) to present a branded experience to your end users.
 
+### Multibrand and custom domains <ApiLifecycle access="ea" />
+
+You can create up to three custom domains with multibrand customizations and up to 200 custom domains by contacting support to increase your limit.
+
+You can only preview or visit a branded page (such as viewing brand assets applied to the Okta-hosted sign-in page) once you map to a custom domain. After you create a brand, map it to a custom domain. Then you can make further customizations, preview them, and publish them.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/custom-url-domain/main/index.md:55]: this is that same paragraph I commented on in the other doc. Not sure what you did to remedy it...so repeat here?

```suggestion
You can only preview or visit a branded page (such as viewing brand assets applied to the Okta-hosted sign-in page) after you map to a custom domain. After you create a brand, map it to a custom domain. Then you can make further customizations, preview them, and publish them.
```

File: packages/@okta/vuepress-site/docs/guides/custom-url-domain/main/index.md [hunk 7]
```diff
@@ -93,8 +101,10 @@ This method of configuring a custom domain is recommended because Okta manages y
 >
 >  * If you have an Okta-managed certificate and you later get a CAA record, Okta can't renew your certificate. You must either add letsencrypt.org to the issuers list or remove the CAA record.
 
-1. In the Admin Console, go to **Customizations** > **Domain**.
-2. In the **Custom URL Domain** box, click **Edit**.
+> **Note:** If you've enabled [Early Access (EA) multibrand customization](https://help.okta.com/okta_help.htm?type=oie&id=csh-branding), your Admin Console navigation is different. See parenthetical notes.
+
+1. In the Admin Console, go to **Customizations** > **Domain**. (EA users: go to **Customizations** > **Brands**, and then select the brand you want.)
+2. In the **Custom URL Domain** box, click **Edit**. (EA user: on the **Domains** tab in the **Custom domain** section, click **Add domain**).
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/custom-url-domain/main/index.md:107]:
```suggestion
2. In the **Custom URL Domain** box, click **Edit**. (EA users: on the **Domains** tab in the **Custom domain** section, click **Add domain**).
```

File: packages/@okta/vuepress-site/docs/guides/custom-url-domain/main/index.md [hunk 8]
```diff
@@ -291,25 +301,21 @@ A custom email address allows you to present a branded experience to your end us
...
-2. Click the **Sender:** link (**Okta <noreply@okta.com>**) near the top of the page.
-
-3. On the Configure Email Sender dialog box, select **Custom email domain** as the type of sender that you want to send system notification emails from.
+## Configure a custom email address
 
-4. In the **Email address to send from**, enter the email address that you want to send the system notification emails from. This is what displays in the emails sent to your users.
+1. In the Admin Console, go to **Customizations** > **Emails**. (EA users: go to **Customizations** > **Brands**, and then select the brand you want. In the **Domains** tab, in the **Email** section, click **Edit** then click **Add custom email domain**).
 
-5. Enter the **Name of sender**. This name appears as the sender in the emails sent to your users.
+2. In the **Email address** field, enter the email address that you want to send the system notification emails from. This is what displays in the emails sent to your users.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/custom-url-domain/main/index.md:310]:
```suggestion
2. In the **Email address** field, enter the email address that you want to send the system notification emails from. This is what appears in the emails sent to your users.
```

File: packages/@okta/vuepress-site/docs/guides/custom-widget/main/index.md [hunk 9]
```diff
@@ -50,14 +50,16 @@ The **Custom Sign-In Page** offers basic and advanced customization options to c
 
 To access this page:
 
-In the Admin Console, go to **Customizations** > **Branding** then click **Edit** in the Sign-in page section.
+> **Note:** If you've enabled [Early Access (EA) multibrand customization](https://help.okta.com/okta_help.htm?type=oie&id=csh-branding), your Admin Console navigation is different. See parenthetical notes.
+
+In the Admin Console, go to **Customizations** > **Branding** then click **Edit** in the Sign-in page section. (EA users: go to **Customizations** > **Brands**, and then select the brand you want.)
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/custom-widget/main/index.md:55]:
```suggestion
In the Admin Console, go to **Customizations** > **Branding** then click **Edit** in the **Sign-in page** section. (EA users: go to **Customizations** > **Brands**, and then select the brand you want.)
```

File: packages/@okta/vuepress-site/docs/guides/custom-url-domain/main/index.md (hunk 6 above)
Comment (brentschaus-okta) [reply packages/@okta/vuepress-site/docs/guides/custom-url-domain/main/index.md]: fixed. or re-fixed.

**Pull request merged**